### PR TITLE
issue305: to support the latest version 2.1.1 schema

### PR DESCRIPTION
**Pull request by sunil-kumar-meena**
Fixes: https://github.com/openBackhaul/TypeApprovalRegister/issues/305

Also Implemented services under FileProfile

Reviewer @PrathibaJee

### CODE REVIEW

File: server/index.js [hunk 1]
```diff
@@ -28,6 +28,6 @@ http.createServer(app).listen(serverPort, function () {
 });
 
 //setting the path to the database 
-global.databasePath = './database/load.json'
```
Comment (IswaryaaS) [inline server/index.js:31]: kindly update the port of TAR to the one which is mapped for 2.1.0 . 
Also, please update in dockerFile as per documentation.

File: server/service/FileProfileService.js [hunk 2]
```diff
@@ -102,71 +97,15 @@ exports.getFileProfileOperation = function(url) {
...
-exports.putFileProfileFilePath = function (url, body) {
-  return new Promise(async function (resolve, reject) {
-    try {
-      console.log(body);
-      await fileOperation.writeToDatabaseAsync(url, body, false);
-      resolve();
-    } catch (error) {
-      reject();
-    }
+exports.putFileProfileFileName = function(body,uuid) {
```
Comment (IswaryaaS) [inline server/service/FileProfileService.js:106]: For file-profile, we can either create a separate issue for implementing the changes. or we can update the code here in the same issue and merge it to code. 
Kindly let me know your views.

**Review by IswaryaaS — CHANGES_REQUESTED** [review]

File: server/index.js (hunk 1 above)
Comment (sunil-kumar-meena) [reply server/index.js:31]: Done

File: server/service/FileProfileService.js (hunk 2 above)
Comment (sunil-kumar-meena) [reply server/service/FileProfileService.js]: implemented FileProfile in this issue only

**Review by IswaryaaS — APPROVED** [review]

**Pull request merged**